packages/core/src/odata/common/request/odata-batch-request-config.ts

```diff
@@ -1,22 +1,32 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
+import { v4 as uuid } from 'uuid';
 import { ODataRequestConfig } from './odata-request-config';
 
 export class ODataBatchRequestConfig extends ODataRequestConfig {
+  /**
+   * @deprecated Since v1.29.0.
+   */
   static readonly content_type_prefix = 'multipart/mixed; boundary=batch_';
 
+  /**
+   * @deprecated Since v1.29.0. Use [[boundary]] instead.
+   */
+  get batchId() {
+    return this.boundary;
+  }
+
   /**
    * Creates an instance of ODataBatchRequestConfig.
    *
    * @param defaultServicePath - The default OData service path
-   * @param batchId - The batch id for building the header and the payload.
+   * @param boundary Request boundary for separation of subrequests. Defaults to an autogenerated value.
```

Comment:
typo?

Reply:
Where? ;)

Reply:
I guess just the `-` is missing. ` boundary - Request boundary for se...`

Reply:
Done